platform-api/che-core-api-infrastructure-local/src/main/java/org/eclipse/che/api/local/LocalWorkspaceDaoImpl.java

```diff
@@ -42,10 +42,9 @@
  * <p>{@link #loadWorkspaces() Loads} & {@link #saveWorkspaces() stores} in memory workspaces
  * to/from filesystem, when component starts/stops.
  *
- *
- * @implNote it is thread-safe, guarded by <i>this</i> instance
- *
  * @author Eugene Voevodin
+ * @author Dmitry Shnurenko
+ * @implNote it is thread-safe, guarded by <i>this</i> instance
```

Comment:
Can you please change it from ` @implNote it is thread-safe, guarded by <i>this</i> instance` to `<p>it is thread-safe, guarded by <i>this</i> instance` and put it above authors declaration(like it was before). In this case formatter will not affect javadoc content